File: 06_pattern_matching.py
def get_url_conf(host):
    mapping = {
        "www.example-a.dev": "firstApp.urls",
        "Www.example-b.dev": "secondApp.urls",
        "www.example-c.dev": "thirdApp.urls"
    }

    return mapping.get(host, "Sorry, no match")

print(get_url_conf("www.example-a.dev"))
# firstApp.urls

print(get_url_conf("www.example.dev"))
# Sorry, no match

File: 06_pattern_matching.js
function getUrlConf(host) {
    switch (host) {
        case "www.example-a.dev":
            return "firstApp.urls"
        case "www-example-b.dev":
            return "secondApp.urls"
        case "www.example-c.dev":
            return "thirdApp.urls"
        default:
            return "Sorry, no match"
    }
}

console.log(getUrlConf("www.example-a.dev"))
// firstApp.urls

console.log(getUrlConf("www.example.dev"))
// Sorry, no match
